src/cmap/auth/mongodb_oidc/automated_callback_workflow.ts

```diff
@@ -0,0 +1,100 @@
+import { type Document } from 'bson';
+import { setTimeout } from 'timers/promises';
+
+import { type Connection } from '../../connection';
+import { type MongoCredentials } from '../mongo_credentials';
+import {
+  OIDC_VERSION,
+  type OIDCCallbackFunction,
+  type OIDCCallbackParams,
+  type OIDCResponse
+} from '../mongodb_oidc';
+import { AUTOMATED_TIMEOUT_MS, CallbackWorkflow } from './callback_workflow';
+import { type TokenCache } from './token_cache';
+
+/** Must wait at least 100ms between invokations */
+const CALLBACK_DELAY = 100;
```

Comment:
```suggestion
const CALLBACK_DEBOUNCE_MS = 100;
```

Very optional suggestion, I think `debounce` is a bit more unambiguous in terms of what exactly it refers to (and I'm just personally a big fan of adding units to my JS time variables)

Reply:
Updated to suggestion.